dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,148 @@
+# Lit release process
+
+Audience: Lit core contributors.
+
+## Objective
+
+The end result of this document is a release of all unreleased changes on the main branch for all packages in the Lit monorepo.
+
+<blockquote>
+Before releasing, ensure the main branch has been imported and tested within Google.
+</blockquote>
+
+### Tl;dr
+
+1. Approve "Release / Release" GitHub action on the latest commit to release.
+1. Code review the automatically created "Version Packages" PR.
+1. Squash and merge the "Version Packages" PR.
+1. Approve the "Release / Release" workflow on the commit created by "Version Packages" PR. This publishes the new releases to NPM and GitHub.
+
+## Process
+
+1. Find the [commit you want to release](https://github.com/lit/lit/commits/main), and click on the yellow dot (or red cross) that represents GitHub Action checks. There should be a pending Github Action called "Release / Release (push)". Click "Details".
+
+![GitHub Actions UI showing the pending Release action. A link has the text "Review pending deployments"](./images/lit-release-process/release-github-action-review-pending.png)
+
+2. Click on the link "Review pending deployments" which will navigate you to a GitHub Action page for the Release workflow. Then click "Review deployments" and approve the pending deployment.
+
+![GitHub actions UI for the Release showing a "Review deployments" button](./images/lit-release-process/release-github-action-review-pending.png)
+
+![GitHub actions UI modal for approving a pending release](./images/lit-release-process/review-pending-release-modal-ui.png)
+
+3. The GitHub release action will now automatically generate a "Version Packages" PR.
+
+<blockquote>
+If a Version Packages pull request has not been created, an error may have occurred in the GitHub action run. Look at the full logs for root cause.
+
+Possible failures previously encountered: A malformed changest file referencing a package name that doesn't exist.
+The fix was to commit the fixed changeset file to main, and re-run the release process on that new commit with the fixed changeset.
+
+</blockquote>
+
+![A screenshot of a GitHub pull request title "Version Packages"](./images/lit-release-process/version-packages-pr.png)
+
+4. Request code reviews from members of the team familiar with the packages that will be released.
+
+   1. The purpose of this pull request is to:
+      1. Delete all the changeset files, moving their contents into the various Changelog files.
+      1. Bump the `window.litElementVersions` global.
```

Comment:
Technically this also bumps `litHtmlVersions` and `reactiveElementVersions`. Not sure if we should list those for completeness's sake, or just make it general and call it "versions global" instead of calling out `litElementVersions` specifically.

Reply:
Changed to be "versions global"